src/client/java/teammates/client/scripts/DataMigrationForSessions.java

```diff
@@ -26,70 +24,42 @@
  * </ul>
  * </p>
  */
-public class DataMigrationForSessions extends DataMigrationBaseScript<Key<FeedbackSession>> {
+public class DataMigrationForSessions extends DataMigrationWithCheckpointForEntities<FeedbackSession> {
 
     public static void main(String[] args) throws IOException {
         new DataMigrationForSessions().doOperationRemotely();
     }
 
-    /**
-     * {@inheritDoc}
-     */
     @Override
-    protected boolean isPreview() {
-        return true;
+    protected Query<FeedbackSession> getFilterQuery() {
+        return ofy().load().type(FeedbackSession.class);
     }
 
-    /**
-     * {@inheritDoc}
-     */
     @Override
-    protected List<Key<FeedbackSession>> getEntities() {
-        return ofy().load().type(FeedbackSession.class).keys().list();
+    protected boolean isPreview() {
+        return true;
     }
 
-    /**
-     * {@inheritDoc}
-     */
     @Override
-    protected boolean isMigrationNeeded(Key<FeedbackSession> sessionKey) {
-        FeedbackSession session = ofy().load().key(sessionKey).now();
-        try {
-            Field followingCourseTimeZoneField = session.getClass().getDeclaredField("wasFollowingCourseTimeZone");
-            followingCourseTimeZoneField.setAccessible(true);
-            return !followingCourseTimeZoneField.getBoolean(session);
-        } catch (ReflectiveOperationException e) {
-            return true;
-        }
+    protected String getLastPositionOfCursor() {
+        return "";
     }
 
-    /**
-     * {@inheritDoc}
-     */
     @Override
-    protected void printPreviewInformation(Key<FeedbackSession> sessionKey) {
-        // nothing to do
+    protected int getCursorInformationPrintCycle() {
+        return 100;
     }
 
-    /**
-     * {@inheritDoc}
-     */
     @Override
-    protected void migrate(Key<FeedbackSession> sessionKey) {
-        ofy().transact(new VoidWork() {
-            @Override
-            public void vrun() {
-                FeedbackSession session = ofy().load().key(sessionKey).now();
-                ofy().save().entity(session).now();
-            }
-        });
```

Comment:
Any reason to delete this part? The reason for this part, I think, is in the documentation above:
```
  * <p>The actual load/save operation happens in a transaction to prevent possible inconsistencies arising from concurrent
  * modification of the entity, e.g. if a student submits or an instructor modifies the session in the midst of the load/save.
 * </p>
```

Reply:
Oh. the new data migration framwrok already wraps the two checks in a transaction.
https://github.com/TEAMMATES/teammates/blob/f5f345b8169e9a0f677ef4cbc8d1854b621fbd8e/src/client/java/teammates/client/scripts/DataMigrationWithCheckpointForEntities.java#L82-L98